src/qt/addressbookpage.cpp

```diff
@@ -46,12 +47,19 @@ class AddressBookSortFilterProxyModel final : public QSortFilterProxyModel
 
         auto address = model->index(row, AddressTableModel::Address, parent);
 
+    #if (QT_VERSION >= QT_VERSION_CHECK(5, 12, 0))
```

Comment:
slight preference for flipping this around where we're checking if less than qt6, i think this makes the code cleaner and inline with how we've done some other qt6 preparation changes, but not a blocker

Reply:
Current implementation allows to test without Qt 6.